__Tests__/e2e.test.js

```diff
@@ -0,0 +1,19 @@
+const puppeteer = require('puppeteer');
+const URL = "http://127.0.0.1:5501/source/index.html";
+
+describe('basic user flow of the website', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8422468)
```suggestion
describe("basic user flow of the website", () => {
```